src/test/interface/stansummary_test.cpp

```diff
@@ -289,14 +289,15 @@ TEST(CommandStansummary, bad_sig_figs_arg) {
       << "\"" << out.command << "\" failed to quit with an error";
 
   arg_sig_figs = "--sig_figs 101";
+  expected_message = "--sig_figs: Value 101 not in range";
   out = run_command(command + " " + arg_sig_figs + " " + csv_file);
   EXPECT_TRUE(boost::algorithm::contains(out.output, expected_message));
   ASSERT_TRUE(out.hasError)
       << "\"" << out.command << "\" failed to quit with an error";
 }
 
 TEST(CommandStansummary, bad_autocorr_arg) {
-  std::string expected_message = "Bad value for option --autocorr";
+  std::string expected_message = "--autocorr: Number less or equal to 0";
```

Comment:
If I don't enter an integer I get a not-super informative error:

```
bbales2@tadpole:~/cmdstan-develop$ ./bin/stansummary --autocorr=0.1 output2.csv
Could not convert: --autocorr = 0.1
Run with --help for more information.
```

Is there a way to get a better one? If not it's fine.

Reply:
excellent question.
looks like CLI11 "master" branch has additional "TypeValidator" but not yet released.  fuck'em.
could implement a workaround - assign to string, check positive int.  PITA.

Reply:
Skip it is fine with me. `--help` for more information is good enuff.